website/scripts/ros/ros_drivers_state.js

```diff
@@ -23,21 +30,65 @@ function subscribeToDriverDiscovery(){
         if(message.name == '/hardware_interface/mock_gnss'){
             switch (message.status) 
             {
-                case 0: //OFF
+                case GPS_DRIVER_STATUS_OFF: //OFF
                     GPSStatus.style.color = '';//grey
                     break;
-                case 1: //OPERATIONAL
-                    GPSStatus.style.color = '#87b821'; //Green
+                case GPS_DRIVER_STATUS_OPERATIONAL: //OPERATIONAL
+                    GPSStatus.style.color = 'rgb(188, 250, 63)'; //Green
                     break;
-                case 2: //DEGRADED
+                case GPS_DRIVER_STATUS_DEGRADED: //DEGRADED
                     GPSStatus.style.color = '#ff6600'; //Orange
                     break;
-                case 3: //FAULT
+                case GPS_DRIVER_STATUS_FAULT: //FAULT
                     GPSStatus.style.color = '#b32400'; //Red
```

Comment:
Not sure why this is showing as salmon, instead of read.

Reply:
> [PR2.docx](https://github.com/usdot-fhwa-stol/carma-web-ui/files/5218817/PR2.docx)

@maefromm Please see below response.

Reply:
Addressed https://github.com/usdot-fhwa-stol/carma-web-ui/pull/47/commits/acbd2822a01a2dc9334dc1b7b23d3f877e6c4f9b